docs/CONTRIBUTING.md

```diff
@@ -109,6 +109,10 @@ Basic rules for pull requests:
 
 All pull requests will be reviewed by the team.
 
+### Translations
```

Comment:
Let's rename this to `Localization`

Reply:
Can you also update the table of contents?

Reply:
Whoops 😅 Added it now and renamed :)